docs/processing_information.md

```diff
@@ -80,21 +78,21 @@ Mapping to these indices followed the same procedures as for RNA-seq data, inclu
+Any cell barcodes that appeared only in ADT data were discarded.
+Cell barcodes that were present only in the RNA-seq data (i.e., did _not_ appear in the ADT data) were assigned zero counts for all ADTs.
+When cells were [filtered based on RNA-seq content](#filtering-cells) after quantification, the ADT count matrix was filtered to match.
 
-### Processed CITE-seq data
+### Processed ADT data
 
 An ambient profile representing antibody-derived tag (ADT) proportions present in the ambient solution is calculated from the `_unfiltered.rds` object using [`DropletUtils::ambientProfileEmpty()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/ambientProfileEmpty.html).
-This ambient profile, along with negative control information, if present, is then as input to calculate quality-control statistics using [`DropletUtils::cleanTagCounts()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html).
-Cells identified by `DropletUtils::cleanTagCounts()` as having high levels of ambient contamination and/or negative control tags are are removed from all counts matrices present in the `_processed.rds` file.
-The RNA count matrix was filtered to match, such that the same cells are present in all RNA and CITE-seq counts matrices.
+This ambient profile, along with negative/isotype control information, if present, is then used as input to calculate quality-control statistics using [`DropletUtils::cleanTagCounts()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html).
+Cells identified by `DropletUtils::cleanTagCounts()` as having high levels of ambient contamination and/or negative/isotype control tags are indicated for removal within the `_processed.rds` file, but they are not actually removed.
 
-Log-normalized ADT counts are calculated using [median-based normalization](http://bioconductor.org/books/3.16/OSCA.advanced/integrating-with-protein-abundance.html#cite-seq-median-norm), again making use of the baseline ambient profile.
-Importantly, this process may fail if any calculated median size factors have a value of 0, in which case log-normalized ADT counts are not provided in the `_processed.rds` object.
+For all cells that _would be_ retained if `DropletUtils::cleanTagCounts()` filtering were applied, log-normalized ADT counts are calculated using [median-based normalization](http://bioconductor.org/books/3.16/OSCA.advanced/integrating-with-protein-abundance.html#cite-seq-median-norm), again making use of the baseline ambient profile.
+For any cells that would be filtered out as determined by `DropletUtils::cleanTagCounts()`, normalized values are not calculated but are instead assigned `NA` values.
+Importantly, if any calculated median size factors have a value of 0, normalization will fail, and log-normalized ADT counts are not provided in the `_processed.rds` object.
```

Comment:
Does this ever happen with our data? If not, I don't think it deserves an "importantly".
```suggestion
If any calculated median size factors have a value of 0, normalization will fail and log-normalized ADT counts are not provided in the `_processed.rds` object.
```

We might also be able to simplify this to: 
```suggestion
If normalization fails, log-normalized ADT counts are not provided in the `_processed.rds` object.
```

I'm not sure the details of why normalization fails are important to this document.